lib/timeline/fps.dart

```diff
@@ -62,12 +58,16 @@ class FramesChart extends LineChart<FramesTracker> {
       final num height = math.min(dim.y, frame.elapsedMs * pixPerMs);
       x -= 3 * units;
 
-      final String color = frame.elapsedMs > FrameInfo.kTargetMaxFrameTimeMs
-          ? '#f97c7c'
-          : '#4078c0';
+      final String color = _isSlowFrame(frame)
+          ? colorToCss(slowFrameColor)
```

Comment:
nit: move the colorToCss to right where you use the color.

Reply:
done.